app/services/qa/linked_data/authority_url_service.rb

```diff
@@ -24,9 +25,10 @@ def action_validation(action)
             raise Qa::UnsupportedAction, "#{action} Not Supported - Action must be one of the supported actions (e.g. :term, :search)"
           end
 
-          def combined_substitutions(action_config, action, action_request, substitutions, subauthority)
+          def combined_substitutions(action_config, action, action_request, substitutions, subauthority, language) # rubocop:disable Metrics/ParameterLists
             substitutions[action_request_variable(action_config, action)] = action_request
-            substitutions[action_subauth_variable(action_config)] = action_subauth_variable_value(action_config, subauthority) if subauthority.present?
+            substitutions[action_subauth_variable(action_config)] = action_subauth_variable_value(action_config, subauthority) if supports_subauthorities?(action_config) && subauthority.present?
+            substitutions[action_language_variable(action_config)] = language_value(language) if supports_language_parameter?(action_config) && language.present?
```

Comment:
`lang` used to be passed in as part of the `substitutions`. This was an artifact of `lang` failing to be removed in the controller.  Now that it is properly removed, it has to be passed in by name.  This is cleaner as it makes the language processing more visible.